widgets/topology/src/widget.tsx

```diff
@@ -145,7 +145,14 @@ Stage.defineWidget({
                 layout
             },
             icons
-        } = _.isEmpty(data) ? { rawBlueprintData: { data: { id: '' }, layout: {} } } : data;
+        } = _.isEmpty(data)
+            ? {
+                  blueprintDeploymentData: {},
```

Comment:
If I'm not mistaken, the destructing above takes `processedBlueprintData` from `data` and extracts it as a variable `blueprintDeploymentData`. Thus, should it not be named `processedBlueprintData`?

```suggestion
                  processedBlueprintData: {},
```

Reply:
Good catch. Suggestion applied.